modules/oadp-backing-up-applications-restic.adoc

```diff
@@ -0,0 +1,33 @@
+// Module included in the following assemblies:
+//
+// * backup_and_restore/application_backup_and_restore/backing_up_and_restoring/backing-up-applications.adoc
+
+[id="oadp-backing-up-applications-restic_{context}"]
+= Backing up applications with Restic
+
```

Comment:
@mburke5678 
I updated this module to remove some of the repetition.

There's no way to verify that the Backup was done with Restic until the backup runs. Or maybe by checking the activity of the Restic pods but that would be overkill.

Reply:
@mburke5678 Could you please have a look at this module and see whether it makes sense?